src/openforms/submissions/tokens.py

```diff
@@ -104,4 +104,95 @@ def _num_days(self, dt) -> int:
         return (dt - date(2001, 1, 1)).days
 
 
+class SubmissionTokenGenerator:
```

Comment:
Let's put this one in the `appointments` app - since we require tokens that are valid until the appointment date.

So also the `get_expiry_days` based on a setting is wrong - we can have an exact validity of a token based on the appointment date, let's use that information instead of having a token that's valid even if the appointment has passed or not valid long enough.